src/sentry/api/endpoints/group_tags.py

```diff
@@ -18,37 +14,8 @@ def get(self, request, group):
             environment_id = self._get_environment_id_from_request(
                 request, group.project.organization_id)
         except Environment.DoesNotExist:
-            group_tag_keys = []
-        else:
-            group_tag_keys = tagstore.get_group_tag_keys(group.project_id, group.id, environment_id)
-
-        # O(N) db access
-        data = []
-        all_top_values = []
-        for group_tag_key in group_tag_keys:
-            total_values = tagstore.get_group_tag_value_count(
-                group.project_id, group.id, environment_id, group_tag_key.key)
-            top_values = tagstore.get_top_group_tag_values(
-                group.project_id, group.id, environment_id, group_tag_key.key, limit=10)
-
-            all_top_values.extend(top_values)
-
-            data.append(
-                {
-                    'id': six.text_type(group_tag_key.id),
-                    'key': tagstore.get_standardized_key(group_tag_key.key),
-                    'name': tagstore.get_tag_key_label(group_tag_key.key),
-                    'uniqueValues': group_tag_key.values_seen,
-                    'totalValues': total_values,
-                }
-            )
-
-        # Serialize all of the values at once to avoid O(n) serialize/db queries
```

Comment:
Empirically it seems to make no difference to the number of db queries, whether these are flattened and serialized in a batch, or serialized individually. So for simplicity of code, I have just done them individually in the refactor

Reply:
As discussed in Slack, you're correct, the need for this went away with: https://github.com/getsentry/sentry/commit/ffb686f56398011ba9ca14c8d22151ce1e86003d